### PR TITLE
Feat: Speed up the startup of storaged

### PR DESCRIPTION
**Pull request by luyade**
## What type of PR is this?
- [ ] bug
- [ ] feature
 https://github.com/vesoft-inc/nebula/issues/5456 
- [ ] enhancement

#### Description:
Open the rocksdb engines in parallel to speed up the startup of storaged

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by Sophie-Xie** [comment]
Pls fix the CI, thanks.

**Review by critical27 — APPROVED** [review]
Good job~

File: conf/nebula-storaged.conf.production [hunk 1]
```diff
@@ -82,7 +82,7 @@
 
 ############## rocksdb Options ##############
 # rocksdb DBOptions in json, each name and value of option is a string, given as "option_name":"option_value" separated by comma
---rocksdb_db_options={"max_subcompactions":"4","max_background_jobs":"4"}
+--rocksdb_db_options={"max_subcompactions":"4","max_background_jobs":"4","skip_checking_sst_file_sizes_on_db_open":"true"}
```
[the remark this answers is not in the document]
Comment (luyade) [reply conf/nebula-storaged.conf.production:85]: Yes, we can remove this option, but I don't see there is any harm but benefit if we leave it here. What do you think?

**Review by pengweisong — APPROVED** [review]

File: conf/nebula-storaged.conf.production (hunk 1 above)
[the remark this answers is not in the document]
Comment (critical27) [reply conf/nebula-storaged.conf.production:85]: Ok for me

**Review by critical27 — APPROVED** [review]
Good job~~~ Thx

**Pull request merged**